tests/integration/sugar/test_transaction.py

```diff
@@ -268,9 +263,7 @@ async def test_calculate_payment_fee(self, client):
         )
 
         # AFTER autofilling the transaction fee
-        payment_signed = await safe_sign_and_autofill_transaction(
-            payment, WALLET, client
-        )
+        payment_signed = await autofill(payment, client)
```

Comment:
This no longer is signed, should `payment_signed` be renamed? (Or is this supposed to be signed still?